orcid_hub/templates/macros.html

```diff
@@ -150,7 +150,12 @@
       <ul class="dropdown-menu" role="menu">
         {% for (st, sn) in [
           ("EMP", "Employment"),
-          ("EDU", "Education"),] %}
+          ("EDU", "Education"),
+          ("DST", "Distinction"),
+          ("MEM", "Membership"),
+          ("SER", "Service"),
+          ("QUA", "Qualification"),
+          ("POS", "Invited Position"),] %}
```

Comment:
Need to mark the "affiliation" button active for the rest of affiliation types...

![menu_active](https://user-images.githubusercontent.com/177266/60228891-3fb26000-98e8-11e9-8fd2-dc16439fa7a5.png)